python/graphstorm/config/argument.py

```diff
@@ -748,19 +752,72 @@ def eval_fanout(self):
             # By default use -1 as full neighbor
             return [-1] * self.num_layers
 
+    @property
+    def textual_data_path(self):
+        """ distillation textual data path
+        """
+        if hasattr(self, "_textual_data_path"):
+            return self._textual_data_path
+        return None
+
+    @property
+    def lm_name(self):
+        """ HuggingFace Transformer-based model name for distillation
+        """
+        if hasattr(self, "_lm_name"):
+            return self._lm_name
+        return None
+
+    @property
+    def pre_trained_name(self):
+        """ Name of pre-trained HuggingFace model weights for distillation
```

Comment:
What does "Name of pre-trained HuggingFace model weights" mean?
Isn't it the same as lm_name?

Reply:
It's not the same. lm_name is about the model architecture, and pre_trained_name is about the model weights. For example, for the same model architecture of "DistilBERT", the weights can be distilbert-base-cased or distilbert-base-uncased.

Reply:
So the two arguments: lm_model and pre_trained_name together can identify one model, right? If this is the case, can we use one argument, with a combined value? 
For example, in the yaml file we use
`lm_name: DistiBertModel distilbert-base-uncased` 
We can decide which combinators to use, here I just use a white space as an example. 

Pros of this method is to let users know that they need to give two values for the LM. Cons is that GS need to have a little bit more logic to process the combined values.
Using two arguments could save GS work and avoid potential errors, but not good user experience.

Reply:
My concern is, if we put architecture name and weights name together, it may confuse people for those who don't want to load any pre-trained weights. 

The current way to separate the two arguments makes it more clear for what is what and user can just ignore pre_trained_name if they don't want to load the weight . However, if combine them together by things like ```lm_name: DistiBertModel:distilbert-base-uncased```, user will need to learn how we parser the argument and be confused if they don't want to load pre-trained weights.

@zhjwy9343 what's your opinion on this?

Reply:
This property can be removed